events/2023-06-20-S4PST.md

```diff
@@ -0,0 +1,142 @@
+
+<strong> Emerging Technologies </strong>. One of the major S4PST objectives is the definition of mechanisms able to drive (R&D, collaborative, shepherd, ...) efforts to guarantee that DOE priorities are part of the PST ecosystem. This will help to develop a more robust, functional, and sustainable PST ecosystem. Important DOE Research Priorities, such as performance portability, extreme heterogeneity, and automatic verification, among many others, will help to build a valuable long-term PST ecosystem exceeding the current sustainability capacity to contribute significantly in key scientific milestones and transformational discoveries. While DOE, and ASCR specifically, maintains a very important software stack, it is part of ASCR identity and our mission to continue to propose new ideas and technologies that can be integrated into DOE portfolio. The value and return of investments for software sustainability on emerging technologies such as modern LLVM-based high-productivity languages and ecosystems (e.g. Julia, Rust, Python/Numba) and modern build systems (e.g. Meson) need to be understood in the DOE context in conjunction with their success in the broader field of computing. This understanding will take into consideration previous PST-related efforts, such as the Defense Advanced Research Projects Agency (DARPA) High Productivity Computing Systems (HPCS) program. This is crucial in the convergence of AI + HPC, as AI is driven by different community and business needs not necessarily focusing on the scalability and science mission aspects of DOE HPC. 
+
+
+</p>
+
+<h2>S4PST Team</h2>
+<p>
+<strong>Keita Teranishi (PI)</strong>, Oak Ridge National Laboratory, USA
+</p>
+<p>
+<strong>William F. Godoy (co-PI)</strong>, Oak Ridge National Laboratory, USA
+</p>
+<p>
+<strong>Pedro Valero-Lara</strong>, Oak Ridge National Laboratory, USA
```

Comment:
We can participants in a future PR

Reply:
That is the idea! 
I want to see first what it looks like before I put 20 more names in it.